main/zoe/guide/contracts/atomic-swap.md

```diff
@@ -57,41 +53,55 @@ const {
 } = await zoe.escrow(aliceOfferRules, alicePayments);
 ```
 
-Alice then makes an offer using the escrowReceipt and tries to collect her payout:
+In order for Alice to escrow with Zoe she needs to redeem her invite. Once Alice redeems her invite she will receive a `seat` and a promise that resolves to her payout.
 
 ```js
-swap.makeFirstOffer(escrowReceipt);
-const payments = await payout;
+const { seat: aliceSeat, payout: alicePayoutP } = await zoe.redeem(
+  aliceInvite,
+  aliceOfferRules,
+  alicePayments,
+);
 ```
 
-She then spreads the `instanceHandle` widely. Bob hears about Alice's
-contract and he decides to look up the `instanceHandle` to see if it
-matches Alice's claims.
+Alice then makes the first offer in the swap:
 
 ```js
+const newInviteP = aliceSeat.makeFirstOffer();
+```
+
+She then spreads the invite widely. Bob hears about Alice's
```

Comment:
I think I'm probably contradicting something I wrote in the tests for this contract, but I think we should change this to say that Alice sends the invite directly to Bob.